src/components/simple-table/SimpleTable.tsx

```diff
@@ -111,18 +116,24 @@ export type SimpleTableProps<T extends object> = {
    * Disable hover background color effect on rows
    */
   noHoverBackground?: boolean;
-
+  /**
+   * Component places at the bottom of the table
+   */
   renderFooter?: ReactNode;
   /**
    * @default false
    * Show checkbox on each row, and checkbox for toggling all in header
    */
   selectable?: {
+    /**
+     * If this is true, then onRowClick will call onRowCheckboxChange with the opposite of the current value
+     */
+    anySelected?: boolean;
```

Comment:
anyChecked?? 👀

Reply:
I think selectable is the better nomenclature here.